### DISCUSSION TITLE
sabnzb SQL HELP

### DISCUSSION THREAD

**Discussion by ghost** [sabnzbd/sabnzbd#2941] [General]
am getting this   ERROR message   sabnzb   SQL Command Failed, 

thanks for any help

**Comment by sanderjo** [comment]
"Discussions", so what do you want to discuss ...?

For support questions, go to https://forums.sabnzbd.org/viewforum.php?f=2

**Reply by Safihre** [reply]
Or Reddit, or Discord.

Just not here.

That's why I deleted your previous issue @tinbin1 .

**Comment by thezoggy** [comment]
> am getting this ERROR message sabnzb SQL Command Failed,
> 
> thanks for any help

Please use one of the support channels: [forums](https://forums.sabnzbd.org/), [subreddit](https://www.reddit.com/r/SABnzbd/), [discord ](https://discord.sabnzbd.org/), or [IRC](https://sabnzbd.org/live-chat.html#) for support/questions.

Per github support:
> The **ghost** username is in place to designate accounts that have been deleted by their owners, but that still have content that is visible across the GitHub platform. If you have write permissions to a repository, you can hide, edit, or delete comments from ghost users.
